src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -374,6 +374,12 @@ internal static PragmaWarningState GetPragmaDirectiveWarningState(this SyntaxTre
         {
             return ((CSharpSyntaxTree)tree).GetPragmaDirectiveWarningState(id, position);
         }
+
+        internal static NullableDirective GetNullableDirectiveState(this SyntaxTree tree, int position)
```

Comment:
>GetNullableDirectiveState [](start = 42, length = 25)

It looks like this method is only used in one place. Consider inlining.

Reply:
Done

Reply:
nit: This file looks like it just introduced a newline. Could be further reverted.

---
In reply to: [294535992](https://github.com/dotnet/roslyn/pull/36464#discussion_r294535992) [](ancestors = 294535992)